drivers/modem/Kconfig.ublox-sara-r4

```diff
@@ -112,4 +112,18 @@ config MODEM_UBLOX_SARA_R4_NET_STATUS_PIN
 
 endif # MODEM_UBLOX_SARA_R4_NET_STATUS
 
+config MODEM_UBLOX_SARA_RSSI_WORK
+	bool "Enable RSSI polling work"
+	default n
+	help
+	  u-blox SARA-R4 device is configured to poll for RSSI
+
+if MODEM_UBLOX_SARA_RSSI_WORK
```

Comment:
Same result but preferred is to have `depends on MODEM_UBLOX_SARA_RSSI_WORK` few lines below. It just looks cleaner that way.

Reply:
Done